Documentation/Upgrade/rook-upgrade.md

```diff
@@ -99,7 +99,7 @@ Also update optional resources like Prometheus monitoring noted more fully in th
 
 ## Helm
 
-* The minimum supported Helm version is **v3.2.0**
+* Be sure to update to a [supported Helm version](https://helm.sh/docs/topics/version_skew/#supported-version-skew)
```

Comment:
How about if we move this to the end of this helm section, and change it to a note? It just doesn't seem critical to list first in the section.
```suggestion
!!! note
 Be sure you are using a [supported Helm version](https://helm.sh/docs/topics/version_skew/#supported-version-skew)
```